docs/_sass/elements.scss

```diff
@@ -149,3 +134,25 @@ button {
   border: 1px solid var(--color-secondary-light);
 }
 
+ul,
+ol {
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+  font-size: 1rem;
+  padding-inline-start: 2rem;
+
+  @media (max-width: 37.5em) {
+    // 37.5em is equivalent to 600px
+    font-size: 1.8rem;
+  }
+
+  @media (max-width: 320px) {
+    font-size: 0.8rem;
+    padding-inline-start: 0.5rem;
+  }
+
+  & li {
+    margin: 2px 0;
+  }
+}
```

Comment:
No changes here, correct?

Including any changes outside of changes material to the issue at-hand makes PR review more challenging, risky, and comes with longer merge times. If you're able to remove these changes, that would be helpful.

Reply:
The CSS was conflicting and not given priority , so I moved it down to ensure it applies correctly @leecalcote